test/integration/test.ads.js

```diff
@@ -774,41 +774,66 @@ QUnit.test('shouldPlayContentBehindAd', function(assert) {
   this.player.duration = function() {return Infinity;};
   videojs.browser.IS_IOS = true;
   videojs.browser.IS_ANDROID = true;
+  this.player.ads.settings.liveCuePoints = true;
```

Comment:
true is the default, is it needed to set this?

Reply:
good catch!